drawer/drawer.css

```diff
@@ -0,0 +1,43 @@
+.title{
+    margin: 0ch;
+}
+
+.main{
+    row-gap: 2rem;
+    align-items: center;
+    background-color: var(--component-background);
+}
+
+.drawer{
+    background:white;
+    max-width:12rem;
+    row-gap: 1rem;
+    padding: 0.5rem 1rem;
+    border-radius: 4px;
+    margin: 2rem auto;
+    
+}
+.drawer-body{
+    row-gap:0.4rem;
+}
+
+.item{
+    display:flex;  
+    width: 100%;
+    border-radius: 4px;
+    padding: 0.5rem 0.6rem;
+    align-items: center;
+    
+}
+
+.item:hover{
+    background-color: var(--component-background);
```

Comment:
does the component color itself need to be changed here to match contrast as well?

Reply:
no since color was same i used that. i m going to change the color and merge